Cargo.toml

```diff
@@ -41,20 +40,31 @@ log = "0.4"
 itertools = "0.10"
 hex = { version = "0.4", features = ["std"] }
 zstd = "0.12.0"
+futures = { version = "0.3.25", optional = true }
+
+# Libraries required for with_file_async() implementations
+async-std = { version = "1.12.0", optional = true }
+tokio = {version = "1", optional = true}
+tokio-util = { version = "0.7.4", features = ["compat"], optional = true}
 
 [dev-dependencies]
 rsa = { version = "0.7" }
 rsa-der = { version = "^0.3.0" }
 env_logger = "0.10.0"
 serial_test = "0.9"
-tokio = {version = "1", features = ["full"]}
 reqwest = { version = "0.11.10", features = ["blocking"] }
 
+# Use for testing async files when async-futures enabled
+tokio = {version = "1", features = ["full"]}
+tokio-util = { version = "0.7.4", features = ["compat"]}
 
 [features]
-default = ["signature-pgp","async-tokio"]
+default = ["signature-pgp", "async-futures", "with-file-async-tokio"]
 
 signature-pgp = ["signature-meta", "pgp"]
 signature-meta = []
-test-with-podman = ["signature-meta"]
-async-tokio = ["tokio/fs", "tokio/io-util"]
+test-with-podman = ["signature-meta", "tokio-util", "futures"]
```

Comment:
Shouldn't this require `async-futures` rather than `futures`?

Reply:
Resolved.